cmake/MujocoDependencies.cmake

```diff
@@ -149,24 +149,30 @@ target_include_directories(
 target_compile_options(qhullstatic_r PRIVATE ${MUJOCO_MACOS_COMPILE_OPTIONS})
 target_link_options(qhullstatic_r PRIVATE ${MUJOCO_MACOS_LINK_OPTIONS})
 
-set(tinyxml2_BUILD_TESTING OFF)
-findorfetch(
-  USE_SYSTEM_PACKAGE
-  OFF
-  PACKAGE_NAME
-  tinyxml2
-  LIBRARY_NAME
-  tinyxml2
-  GIT_REPO
-  https://github.com/leethomason/tinyxml2.git
-  GIT_TAG
-  ${MUJOCO_DEP_VERSION_tinyxml2}
-  TARGETS
-  tinyxml2
-  EXCLUDE_FROM_ALL
-)
-target_compile_options(tinyxml2 PRIVATE ${MUJOCO_MACOS_COMPILE_OPTIONS})
-target_link_options(tinyxml2 PRIVATE ${MUJOCO_MACOS_LINK_OPTIONS})
+option(TINYXML2_USE_SYSTEM_PACKAGE "Use the system's tinyxml2" OFF)
```

Comment:
For coherence with existing options (see https://github.com/google-deepmind/mujoco/blob/50de1b3c2c62a42b394167b853cba9813f6b99c7/simulate/cmake/SimulateDependencies.cmake#L29) could it make sense to call this `MUJOCO_USE_SYSTEM_TINYXML2` ?